packages/react-cookie/src/context.ts

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import {CookieManager} from './manager';
+import {UniversalCookies} from './UniversalCookies';
+
+export const CookieContext = React.createContext<
+  CookieManager | UniversalCookies | null | any
```

Comment:
todo: remove any

Reply:
IMO we just want to have an `interface CookieManager {}` that lays out the API that both of these managers use.

Reply:
Yeah that works